core/src/com/unciv/models/ruleset/tile/TileResource.kt

```diff
@@ -22,10 +22,14 @@ class TileResource : NamedStats() {
         stringBuilder.appendLine(this.clone().toString())
         val terrainsCanBeBuiltOnString: ArrayList<String> = arrayListOf()
         terrainsCanBeBuiltOnString.addAll(terrainsCanBeFoundOn.map { it.tr() })
-        stringBuilder.appendLine("Can be found on ".tr() + terrainsCanBeBuiltOnString.joinToString(", "))
-        stringBuilder.appendln()
-        stringBuilder.appendLine("Improved by [$improvement]".tr())
-        stringBuilder.appendLine("{Bonus stats for improvement}: ".tr() + "$improvementStats".tr())
+
+        if (terrainsCanBeBuiltOnString.isNotEmpty()) {
```

Comment:
Conflicts with #4628 - purely FYI. The function is gone in that PR and replaced. I get the reason - I'll adapt #4628 in any case as such checks wouldn't hurt even if this doesn't make it. (actually I have a gut feeling I already had such tuning in before stashing and splitting the 'pedia project - because I tested the thing with some major mods long ago...? Did I miss something?)